### ISSUE TITLE
[SF][A11y][ui5-button] repetitive text in button's aria-label

### ISSUE THREAD

**Issue by 2bno1** [UI5/webcomponents#12398]
### Bug Description

Button has repetitive text in it's aria-label.
An example here:
```html
<ui5-button accessible-name="Add Item">Add</ui5-button>
```
The button has aria-label "Add Add Item". 

### Affected Component

ui5-button

### Expected Behaviour

The expected aria-label should be "Add Item" which is specified with "accessible-name" attribute.

### Isolated Example

https://stackblitz.com/edit/js-cwutue98?file=index.html,package.json

### Steps to Reproduce

1. Goto https://stackblitz.com/edit/js-cwutue98?file=index.html,package.json
2. Observe the aria-label of the "Add" button
3. "Add" button's aria-label is "Add Add Item"

### Priority

None

### UI5 Web Components Version

2.14.0

### Browser

Chrome

### Operating System

macOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by aleksandar-terziev** [comment]
Hello @2bno1 , I am not able to reproduce the described issue using your isolated example. Can you provide the version of Chrome you are using and  a screenshot demonstrating the repetitive text in `aria-label` ? Any additional information will be helpful.

<img width="675" height="328" alt="Image" src="https://github.com/user-attachments/assets/f1f36ea9-05dd-4d6a-9697-3178c36c7b97" />

**Comment by 2bno1** [comment]
Hello @aleksandar-terziev It looks like the installed @ui5/webcomponents is `2.1.0` although I declared `2.14.0` in `package.json`.

<img width="722" height="322" alt="Image" src="https://github.com/user-attachments/assets/4f72aca6-d5e9-4013-8c8a-24e1bacc123b" />

Please try click "Update All to Latest" icon and see the generated page again.

<img width="396" height="155" alt="Image" src="https://github.com/user-attachments/assets/0577bedd-c8e8-42d4-a108-45ab11f3cbaf" />

**Comment by aleksandar-terziev** [comment]
Thanks @2bno1 , I can reproduce the issue after updating the dependencies.

<img width="302" height="159" alt="Image" src="https://github.com/user-attachments/assets/4c6bb2d3-4f10-4eb3-8080-7ecfa1ce113a" />

**Comment by aleksandar-terziev** [comment]
Hello @UI5/ui5-team-ctr-bal , I am forwarding this issue to you. Can you take a look please ?

**Comment by judith9209** [comment]
Brought up this ticket today 2025-10-09 in UI5 monthly planning meeting to ask for priority.

**Comment by judith9209** [comment]
Thanks a lot @unazko  for the code fix!!!
